### PR TITLE
run prettier on all files, address linter warnings, fine tune eslintr…

### PR DESCRIPTION
**Pull request by gozmanyoni**
…c, remove bind(this)

### CODE REVIEW

**Comment by gozmanyoni** [comment]
Wanted to use this as a base for my bmw-hu and did an initial clean-up of quick things i saw that can be addressed.

Btw - future idea, we could create a monorepo of HU interfaces and have shared devices and libraries among them

File: src/main/PiMost.ts [hunk 1]
```diff
@@ -135,26 +112,23 @@ export class PiMost {
...
+        this.interfaces?.[type]?.parseMessage(message)
+      })
     })
   }
   stream(stream: Stream) {
     this.socketMostClient.stream(stream)
   }
 
-  sendMessage(message: SocketMostSendMessage) {
+  sendMessage = (message: SocketMostSendMessage) => {
```
Comment (gozmanyoni) [inline src/main/PiMost.ts:131]: Initialising this way binds it to this from the get go

Comment (rhysmorgan134) [reply src/main/PiMost.ts:131]: ah, amazing, thanks Yoni!

**Comment by rhysmorgan134** [comment]
> Wanted to use this as a base for my bmw-hu and did an initial clean-up of quick things i saw that can be addressed.
> 
> Btw - future idea, we could create a monorepo of HU interfaces and have shared devices and libraries among them

Yeah this would be an amazing way to move forward with it, would just need to be extendable to allow OEM tweaks that they have done. Thanks for the PR too!

**Pull request merged**